ci/build-ibc-chains.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
```

Comment:
One more question I had, as mentioned in the [issue](#32)

> Allow specifying release (e.g. stargate-4) related to #204

To update images and update to a new gaia version, this is the script we'll be running, right?

Reply:
Yes, the `GAIA_BRANCH` needs to be updated for future versions and to build Docker images for that. This script builds the chain files and put it under `/chains/ci/gaia/[RELEASE VERSION]`. Then it builds the Docker images with these files and prompts to push it to Docker Hub